dashboard/test/factories/pd_factories.rb

```diff
@@ -631,6 +631,7 @@
   factory :pd_regional_partner_mini_contact_hash, class: 'Hash' do
     initialize_with do
       {
+        mini: true,
```

Comment:
This was necessary to create the correct email body [here](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/views/pd/regional_partner_contact_mailer/_form_body.haml) -- I'm not sure if a better approach would just be to add [this](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/models/pd/regional_partner_mini_contact.rb#L36) to mini contact's form data by default (instead of just when sending email)?

Reply:
I think it's fine to do it here too, since this method is only used for testing